db/db.go

```diff
@@ -133,6 +133,30 @@ func (db *DB) Use(name string) *Col {
 	return nil
 }
 
+
+// Creates a collection if one doesn't exist and returns its use.
+func (db *DB) ForceUse(name string) *Col {
+	db.schemaLock.RLock()
+	defer db.schemaLock.RUnlock()
+
+	if !db.ColExists(name) {
+		db.Create(name)
+
+		return nil
+	}
+
+	return db.Use(name)
+}
+
+
+// Check if collection exists
+func (db *DB) ColExists(name string) bool {
+	db.schemaLock.RLock()
+	defer db.schemaLock.RUnlock()
```

Comment:
Not sure if this was necessary, but tried to follow same pattern you had elsewhere